tools/configen/configen/configen.py

```diff
@@ -138,13 +133,23 @@ def generate_module(cfg: ConfigenConf, module: ModuleConf) -> str:
     classes_map: Dict[str, ClassInfo] = {}
     imports = set()
     string_imports: Set[str] = set()
-    
-    def_flags: List[DefaultFlag] = []
+
+    flag_type_dict = {
+        "_convert_": "str",
+        "_recursive_": "bool",
+    }
+    def_flags: List[Parameter] = []
     for name, default in module.default_flags.items():
+        try:
+            flag_type = flag_type_dict[name]
+        except RuntimeError:
+            print(f"{name} flag is currently unsupported by configen.")
```

Comment:
```
log.error(f"{name} is not a supported flag")
```
but actually, just raise an exception. no need to continue working if the user config is bad.